user/migrations/0005_auto_20190807_1933.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.3 on 2019-08-07 19:33
```

Comment:
We don't need this migration file.

Reply:
Tests fail because this migration file isn't there.

Reply:
@Samy-33 Nope, they don't. This migration file is in `user` directory, instead of `apps.user`.
There is an exact replica of this file already in `apps.user`.

Reply:
> There is an exact replica of this file already in apps.user

@sk364 I am not able to find the existing file. Can you please share the link?

Reply:
@Samy-33 https://github.com/Velle-log/hs-api/blob/69eaf7606783304630685710fab9fb877952e225/apps/user/migrations/0005_auto_20190808_1501.py

Reply:
This file is in a commit which is a part of this pr

Reply:
Exactly. This file is already there.

Reply:
Sorry. I am unable to understand. 😅
If I remove this file then it will no more be available in this repo.

Reply:
Ohh, sorry. Understood.

Reply:
Ohh, sorry. Understood.